npm/esbuild-wasm/bin/esbuild

```diff
@@ -81,4 +81,4 @@ fs.read = function () {
 };
 
 const argv = ['node', wasm_exec, esbuild_wasm].concat(process.argv.slice(2));
-wrapper(require, require.main, Object.assign(Object.create(process), { argv }), { instantiate });
+wrapper(require, require.main, Object.assign(Object.create(process), { argv }), Object.assign(Object.create(WebAssembly), { instantiate }));
```

Comment:
As of Golang 1.16.0, the generated `run` function now has this check:
```js
			if (!(instance instanceof WebAssembly.Instance)) {
				throw new Error("Go.run: WebAssembly.Instance expected");
			}
```

This seemed like the simplest solution as opposed to patching the generated code like in https://github.com/evanw/esbuild/blob/add2790afd5f72b39c7eea9e48b2022e56f4cfe3/scripts/esbuild.js#L61